src/client/Microsoft.Identity.Client/ManagedIdentity/ServiceFabricManagedIdentitySource.cs

```diff
@@ -45,8 +45,35 @@ public static AbstractManagedIdentity Create(RequestContext requestContext)
             return new ServiceFabricManagedIdentitySource(requestContext, endpointUri, EnvironmentVariables.IdentityHeader);
         }
 
+        internal override HttpClient CreateCustomHttpClient(RequestContext requestContext)
+        {
+            return new HttpClient(CreateHandlerWithSslValidation(requestContext.Logger));
+        }
+
+        internal HttpClientHandler CreateHandlerWithSslValidation(ILoggerAdapter logger)
+        {
+#if NET471_OR_GREATER || NETSTANDARD1_3_OR_GREATER || NET
```

Comment:
```suggestion
#if NET471_OR_GREATER || NETSTANDARD || NET
```

netstandard2.0 is the only one supported by MSAL. I doubt we will ever support netstandard2.1